src/EFCore.PG/Metadata/Internal/IdentitySequenceOptionsData.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Globalization;
+using System.Text;
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Diagnostics;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Metadata.Internal
+{
+    public class IdentitySequenceOptionsData : IEquatable<IdentitySequenceOptionsData>
+    {
+        public static readonly IdentitySequenceOptionsData Empty = new IdentitySequenceOptionsData();
+
+        public long? StartValue { get; set; }
+        public long IncrementBy { get; set; } = 1;
```

Comment:
Why not nullable?

Reply:
This is a very confusing subject... I made StartValue, MinValue and MaxValue nullable because their defaults depend on other values. For example, StartValue defaults to MinValue for ascending sequences, and to MaxValue for descending ones. So it's important to have a null option, for the user to not specify the value.

However, IncrementBy is always 1 by default, so there's no real need for a null value.